jailer/src/chroot.rs

```diff
@@ -34,7 +34,7 @@ pub fn chroot(path: &Path) -> Result<()> {
             null(),
             root_dir.as_ptr(),
             null(),
-            libc::MS_PRIVATE | libc::MS_REC,
+            libc::MS_SLAVE | libc::MS_REC,
```

Comment:
Would be nice to also change `PRIVATE` to `SLAVE` in the comment on line 30 (it's not the most useful comment; basically `pivot_root` did no work with `MS_SHARED` mount points a while ago; dunno if anything changed), and rename the `Error::MountPropagationPrivate` enum variant (used on line 42) to `MountPropagationSlave`.

Reply:
Yes. I missed that. Also the error reported needs to be changed too. Will fix both.

Reply:
Fixed and updated the PR

Reply:
The comment above still says `// Recursively change the propagation type of all the mounts in this namespace to **PRIVATE**, so`